### PR TITLE
Register warden zahid007

### PR DESCRIPTION
**Pull request by code423n4**
Auto-generated PR to register the new warden zahid007

@zahid-007

### CODE REVIEW

**Comment by sksb2180** [comment]
Warden didn't checked out the channel `i-want-to-be-a-warden` yet.

File: _data/handles/zahid007.json
Comment (alizahid-007) [inline _data/handles/zahid007.json:1]: This is my review

**Review by alizahid-007 — APPROVED** [review]

File: _data/handles/zahid007.json
Comment (alizahid-007) [inline _data/handles/zahid007.json:1]: This is 2nd comment

File: _data/handles/zahid007.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "handle": "zahid007",
+  "moralisId": "HjvKhjG4id4gShUHGSzjqlAt",
+  "image": "./avatars/zahid007.jpeg"
+}
```
Comment (alizahid-007) [inline _data/handles/zahid007.json:5]: This is my code

**Review by alizahid-007 — APPROVED** [review]

**Comment by SamusElderg** [comment]
The user exists but has not yet checked into the `i-want-to-be-a-warden` channel

**Pull request closed without merging**